spec/features/data_access/data_access_workflow_spec.rb

```diff
@@ -8,6 +8,62 @@
   non_downloadable_collection_id = 'C179001887-SEDAC'
   non_downloadable_collection_title = '2000 Pilot Environmental Sustainability Index (ESI)'
 
+  test_collection_1 = 'C90762182-LAADS'
+  test_collection_2 = 'C1000000560-NSIDC_ECS'
+  test_collection_3 = 'C1000000561-NSIDC_ECS'
+
+  let(:echo_id) { "4C0390AF-BEE1-32C0-4606-66CAFDD4131D" }
+
+  context "when a user has three or more collections within a project" do
+    before(:all) do
+      load_page :search, project: [test_collection_1, test_collection_2, test_collection_3], view: :project
+      login
+      click_link "Download project data"
+      wait_for_xhr
+    end
+
+    context "when clicking 'FtpPushPull'on the first collection" do
+      before(:all) do
+        within '.access-item-selection:nth-child(1)' do
+          choose 'FtpPushPull'
+          wait_for_xhr
+        end
+      end
+      it "displays the distribution options for the first collection" do
+        page.find('.access-form').should have_content("Distribution Options", wait: 5)
```

Comment:
Wait_for_xhr just wasn't working in in this case.  As you likely know, it's an in-house function and it does seem to work in most cases.  In this case, it wasn't working at all.

I found 'should' waits for elements to show up, up until the default wait time set in capybara, which is 2.  At the default wait time the test was passing about 1 out of 3.  Adding the 'wait' parameter uses that time instead of the default wait time, and increased it to 5.  It never failed at 5, and so I stuck with it.

I'll revisit it today and see if I can learn more why wait_for_xhr doesn't work, and maybe to look at changing it to 'expect' rather than 'should'.  I would also like to get rid of that one 'sleep'  - made me feel guilty to use it (as it should).